ytools/yast2/create_maintenance_branch

```diff
@@ -45,24 +42,36 @@ def modify_rakefile
   raise "Cannot find Rakefile in pwd" unless File.exist?("Rakefile")
 
   lines = File.readlines("Rakefile")
-  conf_line = lines.grep(/Yast::Tasks.configuration do/).first
-  if conf_line
-    conf_var = conf_line[/do \|\s*(\S+)\s*\|/, 1]
-    line_index = lines.index(conf_line)
+  submit_to = "Yast::Tasks.submit_to"
+
+  new_line = "#{submit_to} \"#{TARGET}\"\n"
+  line_index = lines.index {|l| l =~ /#{submit_to}/ }
+  if line_index
+    lines[line_index] = new_line
   else
-    line_index = 2
-    conf_var = "conf"
-    conf_line = "Yast::Tasks.configuration do |#{conf_var}|\n"
-    lines.insert(line_index, conf_line, "end\n")
+    lines.insert(2, new_line, "\n")
   end
 
-  CONF_OPTIONS.each do |key, value|
-    config_line = lines.grep(/#{conf_var}\.#{key}\s*=/).first
-    new_line = "  #{conf_var}.#{key} = #{value.inspect}\n"
-    if config_line
-      lines[lines.index(config_line)] = new_line
+  if !CONF_OPTIONS.empty?
```

Comment:
Well, if we drop this maybe it also makes sense to change the hacky constants based approach and simply provide BRANCH_NAME and TARGET in the command line.